src/Imath/ImathQuat.h

```diff
@@ -267,30 +281,33 @@ IMATH_CONSTEXPR14 inline Quat<T>::Quat (const Quat<S>& q) IMATH_NOEXCEPT : r (q.
     // empty
```

Comment:
Looks like there should be a IMATH_HOSTDEVICE here, too, right? (At line 279 that is. Why won't GitHub allow a comment on that line?)

Reply:
Yes. Will add.